components/server/ee/src/user/stripe-service.ts

```diff
@@ -113,4 +114,19 @@ export class StripeService {
         });
         return session.url;
     }
+
+    async createSubscriptionForCustomer(customerId: string, currency: Currency): Promise<void> {
+        // FIXME(janx): Use configmap.
+        const prices = {
+            EUR: "price_1LAE0AGadRXm50o3xjegX0Kd",
+            USD: "price_1LAE0AGadRXm50o3rKoktPiJ",
+        };
```

Comment:
Could you create an issue for this?

Reply:
Sure! https://github.com/gitpod-io/gitpod/issues/10655